itests/fixtures.py

```diff
@@ -12,15 +11,35 @@
 
 
 def _get_unused_port():
-    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
-        s.bind(('', 0))
-        return s.getsockname()[1]
+    return _get_unused_ports(1)[0]
+
+
+def _get_unused_ports(count):
```

Comment:
I think Karthik fixed the itest failures, and this looks wrong anyway (it allocates the sockets and then closes them, which means that something else could still take the socket before it's opened again). Could you remove this code, or at least put it into a different PR instead of mixing it into this one?